cmd/podman/patch.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "os/exec"
+    "path/filepath"
+    "bytes"
+
+    "github.com/containers/libpod/cmd/podman/libpodruntime"
+    "github.com/containers/libpod/libpod"
+    "github.com/containers/libpod/pkg/rootless"
+    "github.com/pkg/errors"
+    "github.com/containers/libpod/cmd/podman/cliconfig"
+    "github.com/spf13/cobra"
+)
+
+var (
+    patchCommand     cliconfig.PatchValues
+    patchDescription = `Patch containers files with the given patch.`
+
+    _patchCommand = &cobra.Command{
+        Use:   "patch",
+        Short: "Apply a patch to the specified containers",
+        Long:  patchDescription,
+        RunE: func(cmd *cobra.Command, args []string) error {
+            patchCommand.InputArgs = args
+            patchCommand.GlobalFlags = MainGlobalOpts
+            return patchCmd(&patchCommand)
+        },
+        Example: "DEST_FILE SRC_PATCH_FILE [CONTAINERS]",
+    }
+)
+
+func init() {
+        patchCommand.Command = _patchCommand
+        flags := patchCommand.Flags()
+        flags.BoolVarP(
+            &patchCommand.All,
+            "all",
+            "a",
+            false,
+            "Apply the patch to all the running containers")
+        flags.BoolVarP(
+            &patchCommand.Ignore,
+            "ignore-fail",
+            "i",
+            false,
+            "Ignore if the file to patch doesn't exist in a container")
+
+        rootCmd.AddCommand(patchCommand.Command)
+}
+
+// Define the entrypoint command
+func patchCmd(c *cliconfig.PatchValues) error {
+//func patchCmd(c *cli.Context) error {
+    runtime, err := libpodruntime.GetRuntime(&c.PodmanCommand)
```

Comment:
all new commands must use adapter.GetRuntime and implement both local and remote versions of this.

Reply:
btw, hit me up on IRC if ^^^  does not make sense.